src/Utilities/CssConcatenator.php

```diff
@@ -134,36 +142,45 @@ private static function hasEquivalentSelectors(array $selectorsAsKeys1, array $s
             && \count($selectorsAsKeys1) === \count($selectorsAsKeys1 + $selectorsAsKeys2);
     }
 
+    // phpcs:disable Squiz.Commenting.FunctionComment.MissingParamTag
+    // phpcs:disable Squiz.Commenting.FunctionComment.MissingParamName
+    // phpcs:disable Squiz.Commenting.FunctionComment.IncorrectTypeHint
+
     /**
-     * @param \stdClass $mediaRule Object with properties as described for elements of `$mediaRules`.
+     * @param object{
+     *          media: string,
+     *          ruleBlocks: array<int, object{
+     *            selectorsAsKeys: array<string, array-key>,
+     *            declarationsBlock: string
+     *          }>
+     *        } $mediaRule
      *
      * @return string CSS for the media rule.
      */
-    private static function getMediaRuleCss(\stdClass $mediaRule): string
+    private static function getMediaRuleCss(object $mediaRule): string
```

Comment:
In this case, we'd like to limit this to `stdClass`, and not allow instance of any class, don't we?

Reply:
If the parameter is `stdClass`, Psalm complains that it doesn't match `object` in the DocBlock.  I'll see if it's possible to specify that it's both a `stdClass` and what its properties are...

Reply:
Unfortunately, Psalm doesn't recognize `\stdClass{property: value-type}` - it gives an `InvalidDocBlock` error.